whylog/config/__init__.py

```diff
@@ -30,14 +34,21 @@ def _load_rules(self):
     def _load_log_types(self):
         pass
 
+    def _get_parsers_name(self):
+        parsers_name = OrderedSet()
+        for parser in self._parsers:
```

Comment:
`parser` is confusing. As I deduced it is rather parser name, not (whole) parser. And the following `parser_name.add(parser)` looks like adding parser (not parser name).